packages/apollo-engine-reporting-protobuf/reports.proto

```diff
@@ -188,6 +189,10 @@ message ReportHeader {
 	string runtime_version = 8;
 	// eg "Linux box 4.6.5-1-ec2 #1 SMP Mon Aug 1 02:31:38 PDT 2016 x86_64 GNU/Linux"
 	string uname = 9;
+	// eg "current", "prod"
+	string schema_tag = 10;
+	// The hex representation of the sha512 of the schema IDL string
```

Comment:
Isn't the schema hash based on the introspection result and not the IDL?

Reply:
@martijnwalraven yep, this as copy paste from monorepo but its been changed!